src/Misc/layoutbin/en-US/strings.json

```diff
@@ -143,6 +143,7 @@
     "--auth                 Authentication type. Valid options are PAT (Personal Access Token),",
     "                       Negotiate (Kerberos or NTLM), Integrated (Windows default credentials) and",
     "                       ALT (Alternate Credentials)",
+    "--notificationpipename The name of a named pipe in which to output job start/stop information",
```

Comment:
I dont think we should document this because this isn't really an externally supported feature. I could be wrong. Anyone else want to chime in here?